### DISCUSSION TITLE
Display of date/time below the chart

### DISCUSSION THREAD

**Discussion by mbrammer** [kalkih/mini-graph-card#766] [Ideas]
It would be nice to have a date/time display below a chart. This would help to have a better understanding of the displayed data.

**Example:**
![image](https://user-images.githubusercontent.com/2786547/160091168-a754fc83-77fc-49dc-9cbc-6f5b95f0f59f.png)

**Comment by ildar170975** [comment]
Check this: https://github.com/kalkih/mini-graph-card/issues/739
